pkgs/tools/misc/gammy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, qmake, libXxf86vm, wrapQtAppsHook, ... }:
+
+let
+  pname = "gammy";
+  version = "0.9.58";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Fushko";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "02f19b7acrzip4kbfjgqk06xv1c257rq77khpdg5gz0ai6ayvwm8";
+  };
+
+  nativeBuildInputs = [ qmake wrapQtAppsHook ];
+
+  buildInputs = [ libXxf86vm ];
+
+  installPhase = ''
+    install gammy -Dt $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A GUI tool for adjusting pixel brightness/temperature automatically or manually.";
```

Comment:
Can be shortened:

```suggestion
    description = "GUI tool for adjusting pixel brightness/temperature";
```

Reply:
I think the "automatically or manually" bit is important, I managed to shorten it to 64 chars, so it should fit on one line in https://nixos.org/nixos/packages.html